config/config.example.json

```diff
@@ -8,6 +8,23 @@
     },
     "options": {
       "gasPriceOracle": [
+        {
```

Comment:
As long as this is not the actual strategy that gets used in production

Reply:
I just copied and pasted from [here](https://github.com/api3dao/airseeker/pull/454/files) in order to be consistent but I can drop it and just keep sanitizedProviderRecommendedGasPrice and providerRecommendedGasPrice

Reply:
It all depends on who's supposed to run this and where are they going to get the config file from. I'm concerned that it will be like the MerkleFunderDepository deployments and there will be no visibility into things being done incorrectly.

Reply:
Ok. I need to think this thru, please tell me if any of the following is wrong:

- [ ] Nodary and dAPIs Team deploy MerkleFunderDepository contracts to fund their sponsor wallets, EOA, etc
      1. Each team generates their own mnemonic
      2. Each team generates a merkle-funder config.json with their own values inside the `[merkleFunderDepositories](https://github.com/api3dao/merkle-funder/blob/57-recomend-sanitizedProviderRecommendedGasPrice/config/config.example.json#L35)` array
      3. Each team executes `NETWORK=<chainName> yarn deploy:merkle-funder-depositories` on every chain that is listed in the config.json

      These new contracts need to be funded by manager-multisig so I guess we need to copy over the `merkleFunderDepositories` array into a json file in that repo. I don't really see the need to share anything more from the config.json file in this repo. Maybe we can have a new directory structure like:
      /chain/merkle-funder/deployments/current/dapi/merkle-funder-depositories.json
      /chain/merkle-funder/deployments/current/nodary/merkle-funder-depositories.json
      ...
      /chain/merkle-funder/deployments/history/20230718/dapi/merkle-funder-depositories.json
      etc
      
      We could also add a /chain/merkle-funder/scripts folder where we could have a tx generator script to write tx.json files for sending initial funds to these contracts. The EOA used by the worker will also need an initial funding otherwise it won't be able to send txs (not even fund itself).
      
      I think is worth mentioning that the dAPIs team could even deploy Nodary's MerkleFunderDepository contract if EOA address is provided. The owner for all these contracts should be the same anyways (OwnableCallForwarder)
      
- [ ] Nodary and dAPIs Team deploy a `merkleFunder` worker using the same config.json file generated in the previous step
      1. Each team should set up AWS credentials
      2. Each team should have properly set values in a .env file (serveless.yml shouldn't require any specific setting)
      3. Each team should run `yarn sls:deploy --stage <stageName>`
      
      After successful deployment to AWS each team needs to store the serverless.yml, config.json and .env file used. My initial idea was to have a private keybase repo per team where to put these file but I know that this might not be acceptable. So open to ideas on this problem of keeping the EOA mnemonic safe.

Reply:
Looks correct, it's just that there's a lot of work here that's invisible to the board, such as the script that generates the config

Reply:
I've created a couple of issues to keep track of things needed urgently:
https://github.com/api3dao/manager-multisig/issues/217
https://github.com/api3dao/merkle-funder/issues/69
https://github.com/api3dao/merkle-funder/issues/70